src/core_plugins/kibana/public/management/sections/objects/_objects.html

```diff
@@ -48,12 +48,11 @@ <h1 class="kuiTitle">
           class="kuiTab kbn-management-tab"
           ng-class="{ 'kuiTab-isSelected': state.tab === service.title }"
           ng-repeat="service in services"
-          title="{{ service.title }}"
           ng-click="changeTab(service)"
         >
           {{ service.title }}
-          <small>
-          ({{service.data.length}}<span ng-show="service.total > service.data.length"> of {{service.total}}</span>)
+          <small aria-label="{{:: service.data.length + ' of ' + service.total + ' ' + service.title }}">
```

Comment:
To make this fully make sense, I combined the visible number of services, the total, and the kind of service.